docx/oxml/header.py

```diff
@@ -0,0 +1,14 @@
+from . import OxmlElement
+from .xmlchemy import BaseOxmlElement, ZeroOrMore
+
+
+class CT_Hdr(BaseOxmlElement):
+    """
+    ``<w:hdr>``, the container element for the header content
+    """
+    p = ZeroOrMore('w:p', successors=())
+
+    @classmethod
+    def new(cls):
+        header_elm = OxmlElement('w:hdr')
+        return header_elm
```

Comment:
I like the idea of the XML being the sole source of truth about state. Looking forward to seeing how that works.